registries/adapters/dockerhub_adapter.go

```diff
@@ -231,7 +231,14 @@ func (r DockerHubAdapter) loadSpec(imageName string) (*bundle.Spec, error) {
 		return nil, err
 	}
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", token))
-	return imageToSpec(req, fmt.Sprintf("%s/%s:%s", r.RegistryName(), imageName, r.Config.Tag))
+	req.Header.Add("Accept", "application/json")
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	body, err := commonResponseHandler(resp)
```

Comment:
What do we do with the `err`? looks like we're dropping it on the floor.

Reply:
Good spot this was a miss. Perhaps we should look at using errcheck as part of our make check? I can open an issue. It checks that all errors are being handled